Fix bad situation when Comment exactly repeat the Name. Add sub-category.

--- a/data/florence.desktop.in.in
+++ b/data/florence.desktop.in.in
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Encoding=UTF-8
-_Name=Florence Virtual Keyboard
+Name=Florence
 _Comment=Florence Virtual Keyboard
 Exec=florence
 Terminal=false
 Type=Application
 Icon=@ICONDIR@/florence.svg
-Categories=Application;Accessibility;
+Categories=Application;Accessibility;Utility;
 
